types/msg.go

```diff
@@ -110,6 +145,13 @@ type WithdrawMsg struct {
 	Recipient string `json:"recipient,omitempty"`
 }
 
+// StargateMsg is encoded the same way as a protobof [Any](https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto).
+// This is the same structure as messages in `TxBody` from [ADR-020](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-020-protobuf-transaction-encoding.md)
+type StargateMsg struct {
+	TypeURL string `json:"type_url"`
+	Data    []byte `json:"data"`
```

Comment:
Should be `Value` instead as defined in https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/any.proto#L157

Reply:
Thanks for name update